lib/workers/qemu.js

```diff
@@ -59,12 +76,23 @@ module.exports = class QEMUWorker {
       '-smp', '4'
     ])
 
-    childKillEventHandler(this.child, 'exit')
-    childKillEventHandler(this.child, 'uncaughtException')
-    childKillEventHandler(this.child, 'unhandledRejection')
+    this.child.stderr.on('data', (data) => {
+      stderr += data
+    })
+
+    this.child.on('close', () => {
+      console.log(`QEMU error: ${stderr}`)
```

Comment:
Looks good, but maybe worth doing `console.error()` instead? I think that the `close` event also returns the exit code, and if so, lets print it as well

Reply:
ok